.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  root: true,
+  extends: ['plugin:goodeggs/recommended', 'plugin:goodeggs/flowtype'],
+  env: {
+    node: true,
+  },
+  overrides: [
```

Comment:
Looks like this config results in losing the mocha eslint config for the mocha files. Copy this and you should be good: https://github.com/goodeggs/goodeggs-employee-roles/blob/545bb36aaa0a5f92c1f74e2d2ce4b487b8f2d819/.eslintrc.js#L5-L11 (barring any new eslint errors to fix). It's a bit confusing since this repo contains test helpers and we're testing test helpers... but I think those globs will still apply just fine.